docs/how-tos/upload-module.md

```diff
@@ -252,7 +252,7 @@ Run `viam module upload` to upload your custom module to the Viam Registry.
 Specify the path to the file, directory, or compressed archive (with `.tar.gz` or `.tgz` extension) that contains your custom module code:
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-viam module upload --version <version> --platform <platform> <module-path>
+viam module upload --version <version> --platform <platform> <module-path> --tags=<tags>
```

Comment:
note tags is optional; won't matter for most golang modules, matters for pyinstaller, mostly doesn't matter for source-only python, tbd for c++